app/models/vim_performance_tag.rb

```diff
@@ -3,12 +3,17 @@ def self.instances_are_derived?
     true
   end
 
+  # consistent interface with vim_performance_daily
+  def self.find_entries(_ext_options = {})
+    self
```

Comment:
Please add a comment because it's not obvious that `self` is returned to be used as a scope.